pkg/gdrive/main.go

```diff
@@ -93,7 +94,20 @@ func getClient(config *oauth2.Config) *http.Client {
 	tokFile := "token.json"
 	tok, err := tokenFromFile(tokFile)
 	if err != nil {
-		tok = getTokenFromWeb(config)
+		tokenEnv := os.Getenv("GDRIVE_ACCESS_TOKEN")
+		if tokenEnv == "" {
+			tok = getTokenFromWeb(config)
+			saveToken(tokFile, tok)
```

Comment:
Should return the token fetched from web here if the environment variable doesn't exist:
`return config.Client(context.Background(), tok)`